.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
```

Comment:
This is applicable for frontend not for hardhat script?

Reply:
Yes but template should match how we use the repository. Lets say someone wants to report bug, there is not go to -> click -> scroll 

In hardhat we only have running tests or tasks or set of tasks one by one

Reply:
Templates are not general for every repository, right?